STATIC_CODE_CHECKS.rst

```diff
@@ -76,7 +76,7 @@ The current list of prerequisites is limited to ``xmllint``:
 
 Some pre-commit hooks also require the Docker Engine to be configured as the static
 checks are executed in the Docker environment (See table in the
-Available pre-commit checks<#available-pre-commit-checks>`_ . You should build the images
+Available pre-commit checks <#available-pre-commit-checks>`_ . You should build the images
```

Comment:
A "`" is missing in front of "Available..." and this is still not rendered correctly. Maybe you can help fix it here too?

Reply:
oops you're right I was a bit too sloppy in checking that everything is ok